holidayextras/brands
# Paultons bg

#### What does this PR do? (please provide any background)
Currently Paultons background is made up of a large PNG with css linear gradient added behind.  stripping this back down by including the gradient to the png and exporting as a high quality jpg.  221KB > 67KB

#### How can this be tested?
`cd pattern-library && npm link ../brands/paultons`
Go to 'blocks' page to see a nice centralised starry background....ooooo

#### Any tech debt?
all the savings

#### Screenshots / Screencast
There should be minimal difference between current Paultons brand and this.  Colourstops have been included based on CSS gradient colourstops.  

#### What gif best describes how you feel about this work?
![success](https://cloud.githubusercontent.com/assets/3897844/23794499/42ca5cfc-0588-11e7-9c64-6ffa6a547753.gif)



---

**Reviewer**

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Ensured the release has the correct version bump.
- Checked all the work locally on pattern library.
